packages/ra-ui-materialui/src/input/FileInput.stories.tsx

```diff
@@ -60,7 +69,7 @@ export const FullWidth = () => (
 
 export const Disabled = () => (
     <Wrapper>
-        <FileInput source="attachment" disabled>
+        <FileInput source="attachment" inputProps={{ disabled: true }}>
```

Comment:
I don't see the input disabled on my end. Does this work for you?

I can suggest 
```suggestion
        <FileInput source="attachment" options={{ disabled: true }}>
```
which actually works but really does not provide enough visual feedback IMO (but this is beyond the scope of this PR anyway)